integration_tests.xml

```diff
@@ -1,38 +1,37 @@
-            <file>vendor/behat/mink/driver-testsuite/tests/Basic/CookieTest.php</file>
-            <file>vendor/behat/mink/driver-testsuite/tests/Basic/ErrorHandlingTest.php</file>
-            <file>vendor/behat/mink/driver-testsuite/tests/Basic/IFrameTest.php</file>
-            <file>vendor/behat/mink/driver-testsuite/tests/Basic/ScreenshotTest.php</file>
-            <file>vendor/behat/mink/driver-testsuite/tests/Basic/TraversingTest.php</file>
-            <file>vendor/behat/mink/driver-testsuite/tests/Basic/VisibilityTest.php</file>
-            <directory>vendor/behat/mink/driver-testsuite/tests/Form</directory>
-            <directory>vendor/behat/mink/driver-testsuite/tests/Js</directory>
+            <file>vendor/mink/driver-testsuite/tests/Basic/BasicAuthTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/ContentTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/CookieTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/ErrorHandlingTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/IFrameTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/ScreenshotTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/TraversingTest.php</file>
+            <file>vendor/mink/driver-testsuite/tests/Basic/VisibilityTest.php</file>
+            <directory>vendor/mink/driver-testsuite/tests/Form</directory>
+            <directory>vendor/mink/driver-testsuite/tests/Js</directory>
             <!-- The following have been disabled and their respective equals added to Custom driver tests -->
-            <!--<directory>vendor/behat/mink/driver-testsuite/tests/Css</directory>-->
-            <!--<file>vendor/behat/mink/driver-testsuite/tests/Basic/StatusCodeTest.php</file>-->
-            <!--<file>vendor/behat/mink/driver-testsuite/tests/Basic/HeaderTest.php</file>-->
-            <!--<file>vendor/behat/mink/driver-testsuite/tests/Basic/NavigationTest.php</file>-->
+            <!--<directory>vendor/mink/driver-testsuite/tests/Css</directory>-->
+            <!--<file>vendor/mink/driver-testsuite/tests/Basic/StatusCodeTest.php</file>-->
+            <!--<file>vendor/mink/driver-testsuite/tests/Basic/HeaderTest.php</file>-->
+            <!--<file>vendor/mink/driver-testsuite/tests/Basic/NavigationTest.php</file>-->
         </testsuite>
     </testsuites>
 
     <php>
         <var name="driver_config_factory" value="Behat\Mink\Tests\Driver\PhantomJSConfig::getInstance"/>
-        <server name="WEB_FIXTURES_HOST" value="http://127.0.0.1:6789"/>
         <!-- where driver will connect to -->
         <server name="DRIVER_URL" value="http://127.0.0.1:8510/"/>
         <server name="TEMPLATE_CACHE_DIR" value="/tmp/jcalderonzumba/phantomjs"/>
     </php>
 
     <filter>
         <whitelist>
-            <directory>./src/Behat/Mink/Driver</directory>
+            <directory>./src/</directory>
```

Comment:
this whitelist is to make it possible to collect code coverage. The previous rule was not matching anything, which broke the feature